SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/EntityComponentTestUtils.h

```diff
@@ -0,0 +1,176 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#pragma once
+
+#include "SpatialView/EntityComponentTypes.h"
+
+#include <algorithm>
+
+namespace SpatialGDK
+{
+
+namespace EntityComponentTestUtils
+{
+	static const Schema_FieldId EVENT_ID = 1;
```

Comment:
NIT - The namespace probably does the job fine (although namespace collisions have some unexpected rules).
But I'd prefer to keep them in the same namespace as the other stuff but give them a longer name.